src/stream.rs

```diff
@@ -593,16 +593,30 @@ impl SendBuf {
     }
 
     /// Inserts the given slice of data at the end of the buffer.
```

Comment:
Might want to update the description to say that it returns the actual number of elements that were pushed.

Reply:
Yeah, good point, updated now.